packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/02-video-theme.mdx

```diff
@@ -0,0 +1,32 @@
+---
+id: video-theme
+title: Video Theme
```

Comment:
Just `Theme` would be better I think, similar to React

Reply:
Done